README.md

```diff
@@ -142,6 +142,27 @@ webExt.cmd.run({
   });
 ```
 
+If you would like to run an extension on Firefox for Android:
+```js
```

Comment:
Nit, one empty line between the snippet and the paragraph right before it (it doesn't change the markdown rendering, it is just consistent with the other ones that follows)